ewatercycle/forcing.py

```diff
@@ -0,0 +1,341 @@
+from esmvalcore.experimental import get_recipe
+from pathlib import Path
+
+FORCINGS = {
```

Comment:
I'd suggest changing this name to `DATASETS`. That more accurate, since the *actual* forcing data is the *output* of `generate_forcing`. So you could generate forcing data based on either the ERA5 or the ERA-Interim dataset (and possibly many others).